resources/migrations/000_migrations.yaml

```diff
@@ -14262,6 +14262,53 @@ databaseChangeLog:
         - sql: UPDATE metabase_database SET engine = 'presto-jdbc' WHERE engine = 'presto'
       rollback: # nothing to do, since we don't know which drivers used to be presto
 
+  - changeSet:
+      id: v46.00-81
+      author: calherries
+      comment: Added 0.46.0 -- Add database_id to implicit_action
+      changes:
+        - addColumn:
+            tableName: implicit_action
+            columns:
+              - column:
+                  name: database_id
+                  type: int
+                  constraints:
+                    nullable: false
+        - sql:
+            dbms: postgresql,mariadb,mysql
+            sql: >-
+              UPDATE implicit_action i
+              SET database_id = c.database_id
+              FROM action a
+              JOIN report_card c on a.model_id = c.id
+              WHERE i.action_id = a.id;
+        - sql:
+            dbms: h2
+            sql: >-
+              UPDATE implicit_action i
+              SET database_id = (
+                SELECT c.database_id
+                FROM action a
+                JOIN report_card c on a.model_id = c.id
+                WHERE i.action_id = a.id
+              );
+      rollback: # nothing to do, we do not care to preserve actions from 46->45
```

Comment:
I'm confused by what this means. An empty rollback means to do nothing correct? If you omitted this, wouldn't we get the automatic rollback for `addColumn` which would remove this column? Why specify the no-op when we have the actual rollback at hand?

Reply:
I just removed the no-op rollback for the `addColumn` migration. I forgot to remove that when I split the migration into multiple parts to please the linter. The empty rollback definitely needs to be there explicitly for the sql change, otherwise the linter won't pass

Reply:
ah right. i got confused which migration it was on.